packages/tracing/src/browser/metrics.ts

```diff
@@ -327,3 +370,7 @@ export function _startChild(transaction: Transaction, { startTimestamp, ...ctx }
     ...ctx,
   });
 }
+
+function isMeasurementValue(measurement: any): measurement is number {
```

Comment:
Using `is number` should be fine here as it's generic type. We had a lot of issues with env-specific ones though :( https://github.com/getsentry/sentry-javascript/pull/2277

Reply:
Gotcha. I removed the usage of this TS feature.